sdk/core/azure-core-management/pom.xml

```diff
@@ -83,7 +83,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-http-netty</artifactId>
-      <version>1.7.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-http-netty;current} -->
+      <version>1.6.1</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
```

Comment:
Should we use the last core release commit id to create a release branch to use the GA version? I guess @JimSuplizio recommends using `current` for modules belonging to the "same group".

Reply:
Given there are a few wider changes made in `azure-core-management` and downstream libraries, it may be easier on the release process to change this to `dependency` just for release then going back to `current`. I agree that it isn't the most ideal situation.